DM-6019: Added support for device route ref for Thingpark enterprise connections (#4735)
SoftwareAG/c8y-docs: content/change-logs/device-management/actility-agent-3-1-4-support-for-enterprise-connections.md

---
date: 2026-05-15
title: Fix routing issues for pre-existing Actility ThingPark Enterprise devices
product_area: Device management & connectivity
change_type:
    - value: change-QHu1GdukP

>       label: Feature

Reviewer: Shouldn't this be with Fix change type?

---

> date: ''

Reviewer: ```suggestion
date: 2026-05-15
```

The version is already deployed in eu-latest.

> title: Added support for device routing references in ThingPark Enterprise connections

Reviewer: This title misleads because this already works for the devices that were not created in the Thingpark Enterprise and freshly being registered from the Cumulocity tenant.

It did not work for the devices that were pre-existed in the Thingpark Enterprise platform and wanted to be registered in Cumulocity tenant.

My suggestion:

```suggestion
title: Automatic routing setup for pre-existing Actility ThingPark Enterprise devices
```

Reviewer 2: Or how about more technical: "Fix routing issues for pre-existing..."

product_area: Device management & connectivity
change_type:
    - value: change-QHu1GdukP
      label: Feature
component:
    - value: component-TCiiCOknp5
      label: LPWAN
build_artifact:
    - value: tc--fVxjY7du
label: actility-agent
ticket: DM-6019
version: 3.1.4
---

> The Actility Agent now supports device routing for both ThingPark Wireless and Enterprise connections.

Reviewer 3: ```suggestion
The Actility agent now supports device routing for both ThingPark Wireless and Enterprise connections.
```

> When registering a device in Cumulocity that already exists in ThingPark, the agent automatically updates its routing configuration using routing profiles for Wireless connections and route references for Enterprise connections. 

Reviewer 3: ```suggestion
When registering a device in {{< product-c8y-iot >}} that already exists in ThingPark, the agent automatically updates its routing configuration using routing profiles for Wireless connections and route references for Enterprise connections. 
```

> It also creates the necessary routes in ThingPark if they do not yet exist.

Reviewer 3: ```suggestion
Moreover, the agent creates the necessary routes in ThingPark if they do not yet exist.
```

> Moreover, the agent creates the necessary routes in ThingPark if they do not yet exist.

Reviewer: "routing profiles for Wireless connections" and "route references for Enterprise connections" is API specific language and not non-technical user friendly.

My suggestion:

```suggestion
When registering a device in {{< product-c8y-iot >}} that already exists in the connected ThingPark Enterprise account, the agent automatically updates device's routing configuration in ThingPark to include the new route if it does not yet exist and preserve any previously assigned routes.
```
